import java.io.IOException;

public class Main {

	public static void main  (String[] args)  {
	//	ThrowKeyword.throwKeyword();
	//throw new ArithmeticException("hello world");
		//TryWithResourcesDemo.tryWithResourcesDemo();
		//UserDefinedExceptionDemo.userDefinedExcepton();
		try {
			return;
		}
		finally {
			System.out.println("eh");
		}
	}

}
